contrib/caldav/calcurse-caldav.py

```diff
@@ -263,7 +353,10 @@ def push_objects(objhashes, conn, syncdb, etagdict):
 
 def remove_remote_object(conn, etag, href):
     headers = {'If-Match': '"' + etag + '"'}
-    remote_query(conn, "DELETE", href, headers, None)
+    if authmethod == 'basic':
+        remote_query(conn, "DELETE", href, headers, None)
+    elif authmethod == 'oauth2':
+        remote_query(conn, "DELETE", http_host + href, headers, None)
```

Comment:
Same as above.

Reply:
Because of the way *httplib2* takes URI parameters (absolute URI as opposed to a URI path), the href need to be passed along with the entire beginning of the URL e.g. "https://host.name.com" + "/href/". I could rename the variables to be a little more clear. Or this could be eliminated if we replace *http.client* with *httplib2*